cosmwasm/packages/wormhole-bindings/src/query.rs

```diff
@@ -0,0 +1,42 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Binary, CustomQuery, Empty};
+
+#[cw_serde]
+pub struct Signature {
+    // The index of the observer in the observer set.
+    pub index: u16,
+
+    // The signature.
+    pub signature: Binary,
+}
+
+#[cw_serde]
+#[derive(QueryResponses)]
+pub enum WormholeQuery {
+    // Verifies that `data` has been signed by a quorum of observers from `observer_set_index`.
+    #[returns(Empty)]
+    VerifyQuorum {
+        data: Binary,
+        observer_set_index: u32,
```

Comment:
Any reason why we're not calling this `guardian_set_index`?

Reply:
Done.